notebooks/203-meter-reader/203-meter-reader.ipynb

```diff
@@ -0,0 +1,730 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Industrial Meter Reader\n",
+    "This notebook shows how to create a industrial meter reader with OpenVINO Runtime. We use the PaddlePaddle pre-trained model [PPYOLOv2](https://github.com/PaddlePaddle/PaddleDetection/tree/release/2.4/configs/ppyolo) and [DeepLabV3P](https://github.com/PaddlePaddle/PaddleSeg/tree/release/2.5/configs/deeplabv3p) to built-up a multiple inference task pipeline:\n",
+    "\n",
+    "1) Run detection model to find out the meters, and then crop them from the orgin photo.\n",
+    "2) Run segmentation model on these croped meters to get the pointer and scale instance.\n",
+    "3) Find the location of the poiner in scale map.\n",
+    "\n",
+    "<img align='center' src=\"https://user-images.githubusercontent.com/91237924/166137115-67284fa5-f703-4468-98f4-c43d2c584763.png\" alt=\"Task Pipeline\" width=\"1000\"/>\n",
```

Comment:
Please use markdown instead of html. It will scale image automatically to your screen size